minecraft/protocol/packet/actor_pick_request.go

```diff
@@ -13,6 +13,8 @@ type ActorPickRequest struct {
 	// HotBarSlot is the held hot bar slot of the player at the time of trying to pick the entity. If empty,
 	// the resulting spawn egg should be put into this slot.
 	HotBarSlot byte
+	// WithData ...
```

Comment:
I think we can assume this just means whether to pick the entity with entity metadata. Should probably document something like that.